src/players/VAST.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+import vast from 'vast-client'
+import { callPlayer } from '../utils'
+import createSinglePlayer from '../singlePlayer'
+import { FilePlayer } from './FilePlayer'
+
+const MATCH_URL = /^(https?:\/\/)?((bs\.serving-sys\.com)|(pubads\.g\.doubleclick\.net))/i
```

Comment:
we're positive this will always work? they may be using different serving... aren't they xml files? could we use `/\.xml$/i`?